nemo/core/classes/exportable.py

```diff
@@ -117,7 +117,7 @@ def _export(
 
         # Pytorch's default for None is too low, can't pass None through
         if onnx_opset_version is None:
-            onnx_opset_version = 13
+            onnx_opset_version = 16
```

Comment:
could you pls add comments about why 16?

Reply:
There is literally a comment about it two lines above.